packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -111,7 +116,7 @@ export const PaymentForm = ({
     [validator, onPayment, onPaymentError, stripe]
   );
 
-  stripeElementStyles = checkMedia(matchMedia(SMALL_DEVICE_RULE), stripeElementStyles);
+  stripeElementStyles = checkMedia({ matched: matchMedia(SMALL_DEVICE_RULE), stripeElementStyles });
```

Comment:
This looks like a change not related to the zip field

Reply:
This was a typescript checker change